init-interfaces.sh

```diff
@@ -2,6 +2,13 @@
 
 set -ex
 
+is_configured() {
+  if [[ $(grep primary /etc/NetworkManager/system-connections/* | wc -l) -ge 1 && $(grep secondary /etc/NetworkManager/system-connections/* | wc -l) -ge 1 ]]; then
+    echo "interfaces already configured"
+    exit 0
+  fi
+}
```

Comment:
Don't use exit on functions

```suggestion
is_configured() {
  [[ $(grep primary /etc/NetworkManager/system-connections/* | wc -l) -ge 1 && $(grep secondary /etc/NetworkManager/system-connections/* | wc -l) -ge 1 ]]   
}
```